File: app/src/main/java/com/drdisagree/iconify/provider/RemotePrefProvider.java
package com.drdisagree.iconify.provider;

import static com.drdisagree.iconify.common.Resources.SharedXPref;

import com.crossbowffs.remotepreferences.RemotePreferenceFile;
import com.crossbowffs.remotepreferences.RemotePreferenceProvider;
import com.drdisagree.iconify.BuildConfig;

public class RemotePrefProvider extends RemotePreferenceProvider {
    public RemotePrefProvider() {
        super(BuildConfig.APPLICATION_ID, new RemotePreferenceFile[]{new RemotePreferenceFile(SharedXPref, true)});
    }
}

File: app/src/main/java/com/drdisagree/iconify/provider/RemotePrefProvider.kt
package com.drdisagree.iconify.provider

import com.crossbowffs.remotepreferences.RemotePreferenceFile
import com.crossbowffs.remotepreferences.RemotePreferenceProvider
import com.drdisagree.iconify.BuildConfig
import com.drdisagree.iconify.common.Resources

class RemotePrefProvider : RemotePreferenceProvider(
    BuildConfig.APPLICATION_ID,
    arrayOf(RemotePreferenceFile(Resources.SharedXPref, true))
)